.github/workflows/auto-build.yml

```diff
@@ -3,45 +3,46 @@ name: auto-build
 on:
   push:
     branches:
-      - '1.20.1'
+      - 'sc/intermediary-1.20.5-port'
```

Comment:
This should probably still include both branches right now

Reply:
oops forgot to change that